app/presenters/summary/components/saving.rb

```diff
@@ -1,34 +1,43 @@
 module Summary
   module Components
     class Saving < BaseRecord
+      include TypeOfMeansAssessment
+
       alias saving record
 
       private
 
       def answers # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
         [
           Components::FreeTextAnswer.new(
-            :provider_name, saving.provider_name
+            :provider_name, saving.provider_name, show: true
           ),
           Components::FreeTextAnswer.new(
-            :sort_code, saving.sort_code
+            :sort_code, saving.sort_code, show: true
           ),
           Components::FreeTextAnswer.new(
-            :account_number, saving.account_number
+            :account_number, saving.account_number, show: true
```

Comment:
Why the `show: true` ?

Reply:
Because we want them to show even if they are blank in a draft state. (They are required fields)